test/integration/sanSocialVolumeTest.js

```diff
@@ -48,14 +45,4 @@ describe('SAN_SOCIAL_VOLUME', () => {
     const expectedHeaders = ['Date', 'Mentions Count']
     expect(headers).to.deep.equal(expectedHeaders)
   })
-
-  it('returns a record per every day', () => {
```

Comment:
why are some tests removed?

Reply:
Typically there is not enough data for santiment and they begin to fail randomly. That was the case before I changed from santiment to ethereum and it's the same now. Also there is not a lot of value in these tests.

Reply:
even if there is no data fora given day, there should be a record with 0 or null for that day, right?

Reply:
There should be, but when you run it, it only shows the ones for that do not have 0, just tested them out

Reply:
If we leave it at ethereumSlug the test runs just fine though

Reply:
For sanNews it is removed because maybe if you give it a size of 5 and you only have 3 news articles, then the test would fail.

Reply:
@IvanIvanoff Not exaclty
![Screenshot from 2019-07-29 12-15-25](https://user-images.githubusercontent.com/103871/62036528-a7ffa480-b1fa-11e9-8f0f-a858544fc444.png)